.devcontainer/devcontainer.json

```diff
@@ -43,7 +43,7 @@
     "MS-vsliveshare.vsliveshare",
     "VisualStudioExptTeam.vscodeintellicode"
   ],
-  "postCreateCommand": "echo 'skipping Oryx'" // Disable Oryx build which runs npm install and creates a phantom node_modules folder
+  "postCreateCommand": "echo 'skipping Oryx' && cd common/tools/dev-tool && npm install" // Disable Oryx build which runs npm install and creates a phantom node_modules folder
```

Comment:
We opted to not build the repo code here because it might not be in a buildable state (in development branches). We also use rush to build the packages include dev-tool so using npm install could cause confusion/issues when used with rush.

My recommendation is not do this and do `rush update && rush rebuild -t @azure/dev-tool` manually instead.

Reply:
Can the manual step of rush be a script in the root's package.json? Getting rid of the manual steps so that they are documented somewhere would be helpful for contributors.

Reply:
It is documented in the contributing guide: https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#building-our-repository

Reply:
@deyaaeldeen I don't see the reference to adding and using dev-tools. Could you please provide the line number you though covered this material?

Reply:
The link above links to the repository building section. dev-tool is managed by rush, so building our repository will build dev-tool too. Do you think it could be helpful adding a section on how to build individual packages?

Reply:
Yes. There are several steps I'm learning I need as I create PRs and get successful builds. Could I add some instructions?

Reply:
@diberry that would be great! Feel free to update this PR or open a new one.